src/App.css

```diff
@@ -11,11 +11,7 @@
   font-style: normal;
   font-weight: normal;
   src: local("SF Compact Display Regular"),
-<<<<<<< HEAD
     url("./Fonts/sf-compact-display-regular.woff") format("woff");
```

Comment:
Now that we updated font names in the repo, can you change it back to uppercase?